test/e2e/e2e.go

```diff
@@ -248,3 +253,49 @@ func RunE2ETests(t *testing.T) {
 
 	ginkgo.RunSpecsWithDefaultAndCustomReporters(t, "Kubernetes e2e suite", r)
 }
+
+func podFromManifest(filename string) (*api.Pod, error) {
```

Comment:
ah i see, you're moving this into e2e.go because podFromManifest needs biodata?  I understand why, however, this is a new problem, b/c anybody using e2e.go, will now need to have bin datas on the path ?  maybe e2e.go shouldn't be calling podFromManifest?  is there a way this entire thing can be optional ... see comment below.